composer.json

```diff
@@ -27,6 +27,7 @@
     "require": {
         "php": "^7.1",
```

Comment:
You can use PHP version 7.3 here because of #7

Reply:
Done.